src/main/scala/com/mozilla/telemetry/pings/Ping.scala

```diff
@@ -367,11 +367,19 @@ trait SendsToAmplitudeWithEnvironment extends SendsToAmplitude {
 
     ("user_properties" ->
       ("channel" -> meta.normalizedChannel) ~
-        ("app_build_id" -> meta.appBuildId) ~
-        ("locale" -> meta.`environment.settings`.map(_.locale)) ~
-        ("is_default_browser" -> meta.`environment.settings`.map(_.isDefaultBrowser)) ~
-        ("experiments" -> experimentsArray)) ~
-      ("user_id" -> getClientId)
+      ("sample_id" -> meta.sampleId) ~
+      ("app_build_id" -> meta.appBuildId) ~
+      ("app_name" -> meta.appName) ~
+      ("app_version" -> meta.appVersion) ~
+      ("locale" -> meta.`environment.settings`.map(_.locale)) ~
+      ("is_default_browser" -> meta.`environment.settings`.map(_.isDefaultBrowser)) ~
+      ("country" -> meta.geoCountry) ~
+      ("os" -> meta.os) ~
+      ("os_version" -> meta.`environment.system`.map(_.os.name)) ~
+      ("env_build_arch" -> meta.`environment.build`.map(_.architecture)) ~
+      ("experiments" -> experimentsArray)
```

Comment:
Starting to fill out the values requested by @SuYoungHong in https://github.com/mozilla/SAVANT-study-analysis/blob/master/AMP_CLIENTS.md

It's not clear to me how to get at properties outside of meta (memoryMB, for example, is contained in system). Will chat with @sunahsuh about approaches.